homeassistant/components/august/subscriber.py

```diff
@@ -43,12 +43,17 @@ def _unsubscribe() -> None:
     async def _async_refresh(self, time: datetime) -> None:
         """Refresh data."""
 
+    @callback
+    def _async_scheduled_refresh(self, now: datetime) -> None:
```

Comment:
Should there be an eager keyword for our listeners, or maybe a decorator for the functions we pass in ?

Reply:
I was thinking decorate coros like we do with `@callback` but I think after the initial flood of these we won't have that many.  If I turn out to be wrong on that, I think I'll switch gears and add the decorator, but I don't want to put more checks in the HassJob path if it's not going to be more generally used